app/routers/weight.py

```diff
@@ -0,0 +1,65 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def weight_form(
+        request: Request,
+        session: Session = Depends(get_db),
+        target: Union[float, None] = None,
+        current_weight: Union[float, None] = None,
+        ):
+    user = session.query(User).filter_by(id=1).first()
+    target = user.target_weight
+    if current_weight:
+        return RedirectResponse(url='/')
+    return templates.TemplateResponse("weight.html", {
+        "request": request,
+        "target": target,
+        "current_weight": current_weight,
+        }
+    )
+
+
+@router.post("/weight")
+async def weight(
```

Comment:
Same comment here.

Reply:
This one I prefer leaving the name as it is.

Reply:
because?